lib/kapost_deploy/plugins/notify_honeybadger_after_promote.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require "kapost_deploy/heroku/app_releases"
+
+module KapostDeploy
+  module Plugins
+    # After-promotion plugin to notify honeybadger after a promotion is complete.
+    # Honeybadger.io can be configured to clear all honeybadgers upon this deploy notification.
+    class NotifyHoneyBadgerAfterPromote
+      def initialize(config,
+                     ahead_releases: KapostDeploy::Heroku::AppReleases.new(config.app, token: config.heroku_api_token))
+        self.config = config
+        self.git_config = config.options.fetch(:git_config, nil)
+        self.ahead_releases = ahead_releases
+      end
+
+      def before
```

Comment:
Yes, based on this https://github.com/kapost/kapost_deploy/blob/master/lib/kapost_deploy/task.rb#L123 the method would be called if not defined.

I will remove.